nodewatcher/modules/devices/tplink/cpe.py

```diff
@@ -3,13 +3,13 @@
 from nodewatcher.core.generator.cgm import base as cgm_base, protocols as cgm_protocols, devices as cgm_devices
 
 
-class TPLinkCPE210(cgm_devices.DeviceBase):
+class TPLinkCPE210v1(cgm_devices.DeviceBase):
     """
-    TP-Link CPE210 device descriptor.
+    TP-Link CPE210 v1 device descriptor.
     """
 
-    identifier = 'tp-cpe210'
-    name = "CPE210"
+    identifier = 'tp-cpe210-v1'
+    name = "CPE210 v1"
```

Comment:
Inconsistent naming, check other device descriptors (e.g., it should be in parenthesis, `(v1)`).

Reply:
Yes, you are correct.
I will change it today.